src/main/java-templates/org/postgresql/Driver.java

```diff
@@ -411,7 +411,7 @@ public Connection getResult(long timeout) throws SQLException {
      * @throws SQLException if the connection could not be made
      */
     private static Connection makeConnection(String url, Properties props) throws SQLException {
-        return new @JDBCCONNECTCLASS@(hostSpecs(props),
+        return new ${JDBCCONNECTCLASS}(hostSpecs(props),
```

Comment:
@lordnelson You're right with all what you say, but I would love to get some numbers how many people will really be angry with us adding new driver names in a *major* release of the driver. I mean, all the existing docs saying "org.postgresql.Driver" in the end talk about already EXISTING versions, while we talk about a FUTURE version. I think we overestimate the negative effect of such a change and should not be reluctant just by fear.